hudi-utilities/src/main/java/org/apache/hudi/utilities/UtilHelpers.java

```diff
@@ -285,6 +285,24 @@ private static SparkConf buildSparkConf(String appName, String defaultMaster, Ma
     return SparkRDDWriteClient.registerClasses(sparkConf);
   }
 
+  private static SparkConf buildSparkConf(String appName, Map<String, String> additionalConfigs) {
+    final SparkConf sparkConf = new SparkConf().setAppName(appName);
+    sparkConf.set("spark.ui.port", "8090");
+    sparkConf.setIfMissing("spark.driver.maxResultSize", "2g");
+    sparkConf.set("spark.serializer", "org.apache.spark.serializer.KryoSerializer");
+    sparkConf.set("spark.hadoop.mapred.output.compress", "true");
+    sparkConf.set("spark.hadoop.mapred.output.compression.codec", "true");
+    sparkConf.set("spark.hadoop.mapred.output.compression.codec", "org.apache.hadoop.io.compress.GzipCodec");
+    sparkConf.set("spark.hadoop.mapred.output.compression.type", "BLOCK");
```

Comment:
@Neuw84 are these particularly for AWS Glue?  Could we also update the Hudi website / docs on how Deltastreamer can be used on serverless platform?

Reply:
Hi @yihua,

I just copied them from the other constructor. Most of them won´t have effect on Glue as the change of internal configs is very restricted. 

Will try to make that happen, I was thinking on posting on the AWS blog but it will good to post this also on Hudi website/docs.  

Will work on the Hudi website next. 

Thanks for your inputs!